k2/python/csrc/torch/fsa.cu

```diff
@@ -61,6 +61,20 @@ static void PybindFsaBasicProperties(py::module &m) {
                kFsaPropertiesMaybeCoaccessible;
       },
       py::arg("properties"));
+  m.def(
```

Comment:
Please also update the copyright info of this file and the remaining files.

Reply:
Added, thanks.